src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskService.kt

```diff
@@ -62,84 +62,95 @@ class RiskService(
     }
   }
 
+  private suspend fun handleFetchRiskSummary(crn: String): RiskSummaryResponse? {
+    return try {
+      getValue(arnApiClient.getRiskSummary(crn))
+    } catch (e: WebClientResponseException.NotFound) {
+      log.info("No Risk Summary available for CRN: $crn - ${e.message}")
+      null
+    }
+  }
+
   private suspend fun handleFetchMappaApiCall(crn: String): Mappa? {
-    return try { fetchMappa(crn) } catch (e: WebClientResponseException.NotFound) {
+    return try {
+      fetchMappa(crn)
```

Comment:
I'm probably reading it wrong but it looks like there's a call out to get mappa, if that 404's, the response to UI should still be a 200 with personal details?

Reply:
Yes, that's what will happen 'fetchMappa(..)' here is inside a try/catch block which will swallow any 404 and allow the code execution to proceed with a 200.

Reply:
👍